python/invoke_release/tasks.py

```diff
@@ -1045,6 +1049,67 @@ def _post_rollback(current_version, rollback_to_version):
         plugin.post_rollback(ROOT_DIRECTORY, current_version, rollback_to_version)
 
 
+def _get_version_to_bump(changelog_message):
```

Comment:
"get version level"?

Reply:
`_get_suggested_version_if_any`

Reply:
I went with a mix of the three `_get_version_element_to_bump_if_any`, because "element" is how each section are referred to in https://semver.org/#spec-item-2